n/node-assert-plus/node-assert-plus_ubi_9.3.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : node-assert-plus
+# Version       : master
+# Source repo   : https://github.com/TritonDataCenter/node-assert-plus
+# Tested on     : UBI:9.3
+# Language      : Javascript
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=node-assert-plus
+PACKAGE_VERSION=master
```

Comment:
@vinodk99 make change to accept version as parameter if in future they release some version